app/ts/background/windows/interceptorAccess.ts

```diff
@@ -211,12 +217,12 @@ async function resolve(simulator: Simulator, websiteTabConnections: WebsiteTabCo
 				activeAddress: accessReply.requestAccessToAddress,
 			})
 		}
-		if (request !== undefined) await handleContentScriptMessage(simulator, websiteTabConnections, request, website, activeAddress)
 	}
 	
-	const pendingRequests = await updatePendingAccessRequests(async (previousPendingAccessRequests) => {
-		return previousPendingAccessRequests.filter((x) => !(x.website.websiteOrigin === website.websiteOrigin && (x.requestAccessToAddress?.address === accessReply.requestAccessToAddress || x.requestAccessToAddress?.address === accessReply.originalRequestAccessToAddress)))
-	})
+	const isAffectedEntry = (pending: PendingAccessRequest) => pending.website.websiteOrigin === website.websiteOrigin && (pending.requestAccessToAddress?.address === accessReply.requestAccessToAddress || pending.requestAccessToAddress?.address === accessReply.originalRequestAccessToAddress) 
+
+	const pendingRequests = await updatePendingAccessRequests(async (previousPendingAccessRequests) => previousPendingAccessRequests.filter((pending) => !isAffectedEntry(pending)))
+	const affected = pendingRequests.previous.filter((pending) => isAffectedEntry(pending))
```

Comment:
Can this be moved down to line 234, or does it need to happen prior to the two conditionals?